Parsers/ASimWebSession/Test/PaloAlto_CortexDataLake_ASimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,99 @@
+Result
+"(1) Warning: Missing recommended field [ASimMatchingIpAddr]"
+"(2) Info: Missing optional alias [Duration] aliasing non-existent column [NetworkDuration]"
+"(2) Info: Missing optional alias [Hash] aliasing non-existent column [MD5|SHA1|SHA256|SHA512]"
+"(2) Info: Missing optional alias [InnerVlanId] aliasing non-existent column [SrcVlanId]"
+"(2) Info: Missing optional alias [OuterVlanId] aliasing non-existent column [DstVlanId]"
+"(2) Info: Missing optional alias [UserAgent] aliasing non-existent column [HttpUserAgent]"
+"(2) Info: Missing optional field [AdditionalFields]"
+"(2) Info: Missing optional field [DstAppId]"
+"(2) Info: Missing optional field [DstAppName]"
+"(2) Info: Missing optional field [DstAppType]"
+"(2) Info: Missing optional field [DstBytes]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DstDeviceType]"
+"(2) Info: Missing optional field [DstGeoCity]"
+"(2) Info: Missing optional field [DstGeoLatitude]"
+"(2) Info: Missing optional field [DstGeoLongitude]"
+"(2) Info: Missing optional field [DstGeoRegion]"
+"(2) Info: Missing optional field [DstInterfaceGuid]"
+"(2) Info: Missing optional field [DstInterfaceName]"
+"(2) Info: Missing optional field [DstOriginalUserType]"
+"(2) Info: Missing optional field [DstPackets]"
+"(2) Info: Missing optional field [DstUserId]"
```

Comment:
DestinationUserID coming from source?

Reply:
why we are not mapping all fields available in logs?

Reply:
Done.
We have rechecked the data & fields available in document and mapped all possible fields.